### DISCUSSION TITLE
Filament tray, nozzle holder and tool holder.

### DISCUSSION THREAD

**Discussion by SteveBz** [1coderookie/KobraGoNeoInsights#5] [Show and tell]
Here are three other Neo Mods I've implemented:

**Nozzle holder**

https://www.thingiverse.com/thing:4962378

![WhatsApp Image 2023-03-06 at 08 39 33](https://user-images.githubusercontent.com/1646078/223064489-b040112c-6b13-41a8-b8ad-430fbbae9b3e.jpeg)

**Filament tray**

https://www.thingiverse.com/thing:5880981

Very hard to remove the supports and to fit.  Slow and easy does it and needs a bit of force at the end.  I used a clamp.

![WhatsApp Image 2023-03-06 at 08 39 13](https://user-images.githubusercontent.com/1646078/223064462-bc586498-14b5-4343-9a35-8f3b43a161f9.jpeg)

**Tool holder.**

https://www.thingiverse.com/thing:5818938

Also hard to fit.  You can't hold the T-Nuts behind the holder.  In the end I only used 2 bolts, but that's very solid.

![WhatsApp Image 2023-03-06 at 08 38 59](https://user-images.githubusercontent.com/1646078/223064423-e634c571-6c84-4134-a035-f89c32a8dca1.jpeg)

**Comment by 1coderookie** [comment]
Thanks, looks pretty cool with the black filament :)
I also saw them and thought about mentioning them, but I wasn't sure if I should. Would you recommend printing them? 
Especially the nozzle holder seems kinda nonsensical to me as I personally would never store my spare nozzles in an exposed area like this, I put them in little cans for safer storage.
The tool holder does look kinda useful, but I'd think that it vibrates too much during printing. Also it might be in the way when I need to tinker and fix anything again which happens all the time actually..  
And the tray I actually wanted to print before I switched to Klipper, but Klipper positions the nozzle over the front left corner in parking position, so I guess there's no need for me personally to print out that one..

**Comment by 1coderookie** [comment]
"You can't hold the T-Nuts behind the holder." - fyi: there are T-nuts you can screw on, place them in the profile and then they lock themselves up while turning the screw. So you also don't have to remove the cap at the end of a profile to get them inserted, which is also pretty nice.

**Comment by SteveBz** [comment]
My issue is that I keep everything in zip lock bags and shoe boxes, so it's just nice to keep things where they are used. 

I want to upgrade to stainless Steel nozzles, so when I do that I won't need to worry about replacing them. 

Interesting about Klipper. I read the link you sent and it says you lose your screen. Is that not a problem? 

⁣Get BlueMail for Android ​

**Comment by 1coderookie** [comment]
Ah ok - well, mine is standing on a little drawer where I keep everything stored in, so that's quite comfortable actually.  

Remember that you have to use higher print temps when using StainlessSteelNozzles, iirc about 10° higher then you use to print with brass nozzles or so. 

I just opened a discussion thread for Klipper #6 , so that we have everything in place just in case somebody stumbles across it, so let's talk about your question over there..

**Comment by SteveBz** [comment]
I didn't know that about nozzle temps. 

Tx. 

Steve. 

Ps, I don't know your name. 

⁣Get BlueMail for Android ​

**Reply by 1coderookie** [reply]
It's mentioned at my page tho - my name is Ulf :)

**Comment by SteveBz** [comment]
Nice to meet you Ulf.

Steve.

**Reply by 1coderookie** [reply]
Same, Steve :)
